lib/plugin.js

```diff
@@ -111,19 +110,26 @@ export default (ctx, inject) => {
       ? '<%= options.browserBaseURL %>'
       : (process.env._HTTP_BASE_URL_ || '<%= options.baseURL %>')
 
+  const headers = <%= JSON.stringify(options.headers, null, 4) %>
```

Comment:
[editorconfig](https://github.com/nuxt/http/blob/dev/.editorconfig#L5) is set to 2

Reply:
Fixed